users-persistence/src/main/java/engineering/everest/lhotse/users/persistence/PersistableUser.java

```diff
@@ -39,7 +39,7 @@
 
     private boolean disabled;
     @ElementCollection(fetch = EAGER)
-    private Set<Role> roles =  EnumSet.noneOf(Role.class);
+    private Set<Role> roles = new HashSet<>();
```

Comment:
Any reason for replacing `EnumSet` with `HashSet`?

Reply:
A tired mind. I'll revert this.